drivers/i2s/Kconfig.mcux

```diff
@@ -0,0 +1,55 @@
+# MCUX I2S driver configuration options
+
+# Copyright (c) 2020, NXP
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig I2S_MCUX_SAI
+	bool "NXP MCUX I2S controller driver"
+	depends on SOC_SERIES_IMX_RT
+	help
+	  Enable I2S support on the I.MX family of processors.
+
+if I2S_MCUX_SAI
+
+config I2S_0
+	bool "I2S port 0"
```

Comment:
We can get rid of configs `I2S_0`, `I2S_1 `and `I2S_2 `and use the `DT_INST `defines provided by device tree

Reply:
yes, correct use the DT_INST_STASTUS_OKAY instead